includes/admin/notes/class-wc-notes-run-db-update.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * WooCommerce: Db update note.
+ *
+ * Adds a note to complete the WooCommerce db update after the upgrade in the WC Admin context.
+ *
+ * @package WooCommerce
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\Jetpack\Constants;
+use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+
+/**
+ * WC_Notes_Run_Db_Update.
+ */
+class WC_Notes_Run_Db_Update {
+	const NOTE_NAME = 'wc-update-db-reminder';
+
+	/**
+	 * Attach hooks.
+	 */
+	public function __construct() {
+		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
```

Comment:
Added more conditions in f3dceae, checking the loading order.

Reply:
Tested this now by removing the `wp_wc_admin_notes` db table and the code just runs with no ill side effects, with $note_id == 0. The reminder will only show up once the db tables for WC Admin get created, which should happen during the first `admin_init` in `\Automattic\WooCommerce\Admin\Install::check_version` (or during subsequent `admin_init` in case the version of wc admin changes).